members/PAL000460.yaml

```diff
@@ -53,5 +53,4 @@ contact_form:
     - selector: "#js-workaround"      
   success:
     body:
-      contains: "Javascript success message reached"
-      excludes: "Your Message"
+      excludes: "Leaving this box checked verifies that you would like to receive legislative email updates"
```

Comment:
should this also have `contains: Javascript success message reached` since there is a javascript step above submitting the  message and then setting this success message in the body? also, should there be a step to uncheck this checkbox?